### PR TITLE
Public map rendering total of mapviews

### LINKED ISSUE #3756: Public map view should have total mapviews and not only the ones during the last 30 days

### PR DESCRIPTION
**Pull request by xavijam**
Using what @lbosque pushed days ago:

```ruby
Carto::Visualization.where(id: @visualization.id).first.total_mapviews 
```

cc @lbosque @saleiva @javisantana 

Fixes #3756

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3865/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3874/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3875/))

**Comment by xavijam** [comment]
Adding @kartones to this PR.

File: app/controllers/admin/visualizations_controller.rb [hunk 1]
```diff
@@ -222,6 +222,8 @@ def public_map
     @avatar_url             = @visualization.user.avatar
     @google_maps_api_key = @visualization.user.google_maps_api_key
 
+    @mapviews = CartoDB::Visualization::Member.new(id: @visualization.id).fetch.total_mapviews
```
Comment (Kartones) [inline app/controllers/admin/visualizations_controller.rb]: Wait a sec, if there's already a `@visualization` instance, you just need to do `@mapviews = @visualization.total_mapviews` :dancer: 

I didn't saw properly the code around... so even easier

Comment (xavijam) [inline app/controllers/admin/visualizations_controller.rb]: :dancer:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3877/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3879/))

**Comment by xavijam** [comment]
Test passing, tested in staging. Ready to deploy?

**Comment by saleiva** [comment]
+1

**Comment by Kartones** [comment]
+1

**Pull request merged**